pyface/grid/checkbox_image_renderer.py

```diff
@@ -12,4 +12,4 @@
 logger = logging.getLogger(__name__)
 logger.warning("DEPRECATED: pyface.grid, use pyface.ui.wx.grid instead.")
 
-from pyface.ui.wx.grid.checkbox_image_renderer import *
+from pyface.ui.wx.grid.checkbox_image_renderer import *  # noqa: F401
```

Comment:
Is it feasible to not do this? Or are these OK because they're preserving the old interface until removal?

Reply:
i was considering just ignoring these (`pyface/grid/*`) entirely while I do some digging to figure out if/when we can throw out `pyface/grid/` entirely. I'll create a separate issue.

Reply:
and the final issue around deprecation - https://github.com/enthought/pyface/issues/918